git-show-tree: add page (#6164)
tldr-pages/tldr: pages/common/git-show-tree.md

# git show-tree

> > Show a decorated tree graph for each branch, showing commits other statistics.

Author: `git log --all --graph --decorate --oneline --simplify-by-decoration` ( from the repo here https://github.com/tj/git-extras/blob/master/bin/git-show-tree )

> > Show a decorated tree graph for each branch, showing commits other statistics.

Author: Also what about the following 

```suggestion
> Show a decorated tree graph for a Git repository, showing annotations.
```

> > Show a decorated tree graph for each branch, showing commits other statistics.

Reviewer: > `git log --all --graph --decorate --oneline --simplify-by-decoration` ( from the repo here https://github.com/tj/git-extras/blob/master/bin/git-show-tree )

wohoooo .. i was pretty close :)

> > Show a decorated tree graph for each branch, showing commits other statistics.

Reviewer 2: I like the last one, but it's not obvious what kind of tree it is showing. How about:
```suggestion
> Show a decorated tree graph with all branches of a Git repository, showing annotations.
```
